### PR TITLE
[TACHYON-592] Fix Mac OS X HFS+ available provisioning size for ramdisk

### PR DESCRIPTION
**Pull request by qnu**
https://tachyon.atlassian.net/browse/TACHYON-592

### CODE REVIEW

**Comment by calvinjia** [comment]
@qnu Thanks for the patch. With this change, is the ram disk size guaranteed to be greater than or equal to the configured space (for example, if I ask for 4 GB, will I always get >= 4GB)?

**Comment by qnu** [comment]
@calvinjia Yes, as its inline comments noted.

File: bin/tachyon-mount.sh [hunk 1]
```diff
@@ -70,6 +70,64 @@ function mem_size_to_bytes() {
...
+
+  # quota users file and quota groups file
+  quota_users_file_size=$(((n_gb * 256 + 1) * 64))
+  quota_groups_file_size=$(((n_gb * 32 + 1) * 64))
+
+  metadata_size=$((alloc_bitmap_size + ext_overflow_file_size + journal_file_size + \
+    catalog_file_size + hot_file_size + quota_users_file_size + quota_groups_file_size))
+
+  alloc_size=$((request_size + metadata_size))
+  echo $((alloc_size / sector_size))
```
Comment (calvinjia) [inline bin/tachyon-mount.sh]: Does it matter if this rounds down?

Comment (qnu) [inline bin/tachyon-mount.sh]: By my estimation, it does not matter. Previous n_gb, n_kb, etc. calculation has been rounded up, which provides sufficient number for calculating metazone size. Also note that if round-down happens, it always rounds small values in bytes, such as 10, 5, 256, sector_size (512).

**Comment by calvinjia** [comment]
+1 LGTM

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**